Holographic-Oracle-Engine/onyx_pdf_engine.py

```
@@ -21,22 +22,25 @@ def __init__(self):
+            "/usr/share/fonts/truetype/msttcorefonts/Arial.ttf",  # Linux (msttcorefonts)
+            "/Library/Fonts/Arial.ttf",  # macOS
+        ]
+        self.font_name = 'Helvetica'
+        self.mono_font = 'Courier'
+        for path in font_paths:
+            if os.path.exists(path):
+                try:
+                    pdfmetrics.registerFont(TTFont('Arial', path))
+                    self.font_name = 'Arial'
+                    logger.info("Registered TrueType Font: Arial")
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)